content/en/dashboards/functions/rollup.md

```diff
@@ -2,6 +2,10 @@
 title: Rollup
 aliases:
     - /graphing/functions/rollup/
+further_reading:
+- link: "/dashboards/guide/rollup-cardinality-vizualizations"
```

Comment:
Would it be possible to correct the spelling in the file name (and in the links to it) to `visualizations`? I just noticed this 😅

Reply:
Thanks for catching that :)

Reply:
```suggestion
- link: "/dashboards/guide/rollup-cardinality-visualizations"
```